crates/core/tedge/src/cli/init.rs

```diff
@@ -89,6 +89,7 @@ impl TEdgeInitCmd {
         create_directory(config_dir.join("operations").join("c8y"), &permissions).await?;
         create_directory(config_dir.join("plugins"), &permissions).await?;
         create_directory(config_dir.join("sm-plugins"), &permissions).await?;
+        create_directory(config_dir.join("log-plugins"), &permissions).await?;
```

Comment:
This seems to be not aligned with the sudoers file where the log-plugins are expected to be in `/usr/lib/tedge/log-plugins/`

Reply:
Yeah, this is not required anymore as the new location for user plugins (`/usr/lib/tedge/log-plugins`) is specified in the `nfpm` config.

That being said, now it's not fully clear to me as to what all directories belong to the `nfpm` tool and what belongs here. That separation is not fully clear to me.

Reply:
Done in 65ca15a